app/views/reservations/_list_items_in_cart.html.erb

```diff
@@ -5,16 +5,13 @@
     <table id="table_list_items" class="table table-bordered table-striped table-condensed table-hover">
       <% cart.items.each do |model_id, quantity| %>
         <tr>
-          <td class="cart-items-quantity">
-            <%= quantity %>
+          <td id="quantity_col">
+            <%= form_tag url_for(controller: 'catalog', action: 'edit_cart_item', id: model_id), remote: true, class: 'form-vertical', id: 'quantity_form',method: :put do %>
+
+              <%= number_field_tag('quantity', quantity, class: 'autosubmitme form-control quantity', min: 0, step: 1, id: 'quantity_field') %>
```

Comment:
I'm a little concerned that we're using the same id for all of these number fields; it will make writing tests more difficult; it would probably better to use a `quantity_field` class for styling and then a `quantity_field_#{model_id}` id for testing.